absolventa/absolventa_xml.md

```diff
@@ -1,38 +1,93 @@
 # Absolventa XML
 
 ```XML
+<?xml version="1.0" encoding="UTF-8"?>
 <job_offer>
+  <title>Praktikum Seefahrt (w/m/d)</title>
   <mode>premium</mode>
-  <title>Trainee Web Development (m/w)</title>
-  <external_url>https://example.com/jobs/1</external_url>
-  <description><![CDATA[<p>Basic HTML tags are allowed here</p>]]></description>
-  <started_at type="datetime">2017-01-03T00:00:00+02:00</started_at>
-  <ended_at type="datetime">2017-12-10T12:14:42+01:00</ended_at>
-  <!-- You need to either provide an URL or an email address for application. -->
-  <application_method>email/url</application_method>
-  <application_url>https://example.com/jobs/1/apply</application_url>
-  <application_email>application@absolventa.de</application_email>
-  <job_offer_locations>
-    <job_offer_location>
-      <street>Friedrichstraße 67</street>
-      <zip>10117</zip>
-      <city>Berlin</city>
-      <country>Deutschland</country>
-    </job_offer_location>
-    <job_offer_location>
-      <street>Greifswalder Straße 212</street>
-      <zip>10405</zip>
-      <city>Berlin</city>
-      <country>Deutschland</country>
-    </job_offer_location>
-  </job_offer_locations>
-  <contact_name>Rosi Beckers</contact_name>
-  <contact_email>rosi.beckers@absolventa.de</contact_email>
-  <contact_phone>030240483121</contact_name>
-  <contact_position>Leitung Personalwesen</contact_position>
-  <trainee_gefluester>true</trainee_gefluester>
+
+  <application_email>elaine.marley@absolventa.de</application_email>
+  <application_url><![CDATA[https://www.example.com/jobs/seefahrt/apply]]></application_url>
+  <started_at>2019-11-11T13:14:40+01:00</started_at>
+  <ended_at>2019-11-15T23:59:59+01:00</ended_at>
+
+  <city>Berlin</city>
+  <street>Friedrichstraße 67</street>
+  <zip>10318</zip>
+  <country>Deutschland</country>
```

Comment:
Don't we need the former option for passing in multiple locations?
```
<job_offer_locations>
  <job_offer_location>
    <city>Berlin</city>
    <street>Friedrichstraße 67</street>
    <zip>10318</zip>
    <country>Deutschland</country>
  </job_offer_location>
</job_offer_locations>
```

Reply:
Thanks, @dansch - good catch! The missing block has been left out unintentionally and I need to get it on board again - our data structure changes do not touch the management of locations at all!